merkle-js/src/generate_metadata_json.ts

```diff
@@ -0,0 +1,82 @@
+import { merkle, num,cairo } from 'starknet';
+import * as starkCurve from 'micro-starknet';
+import * as fs from 'fs';
+
+// read data from .csv file
+// array of  [address, token_id, task_id, name, rank, score, level, total_eligible_users, proof]
+type Data = [string, number, number, string, number, number, number, number, string[]];
+let list = fs.readFileSync('/home/felix/Downloads/median/output10_proof.txt', 'utf8').split('\n').map(line => {
+    if (line === '') {
+        return null;
+    }
+    const splitLine = line.split(',');
+
+    const proofIndex = splitLine.findIndex((element) => element.includes('['));
+    
+    const proofString = splitLine.slice(proofIndex).join(',').trim();
+    const proofArray = proofString.slice(1, proofString.length - 1).split(',').map(s => s.trim());
+    
+    let data: Data = [splitLine[0], parseInt(splitLine[1]), parseInt(splitLine[2]), splitLine[3], parseInt(splitLine[4]), parseInt(splitLine[5]), parseInt(splitLine[6]), parseInt(splitLine[7]), proofArray];
+    return data;
+});
+
+let dir_prefix = '/home/felix/Downloads/median/ipfs-json/';
```

Comment:
Move this and all such path variables to environment variables. Leave this in a comment as an example.